_docs_integrate/propose-attribute-to-peer.md

```diff
@@ -38,22 +38,23 @@ The Sender wants to propose an Attribute to the Recipient. To do this, the Sende
-| -------------- | --------------- | :-------: | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Identity       | Sender          |     ✗     | `N/A`           | It makes no sense to propose own Attributes, use [ShareAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#shareattributerequestitem) instead.                                                                                                                                                                                           |
-| Identity       | Recipient       |     ✓     | `USER_DECISION` | Company sends name and address to new customer during its onboarding process.                                                                                                                                                                                                                                                                                |
-| Relationship   | Sender          |     ✗     | `N/A`           | It makes no sense to propose own Attributes, use [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) or [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) instead.                                                                               |
-| Relationship   | Recipient       |     ✓     | `USER_DECISION` | With this combination the **Sender asks the Recipient for the one-time permission** to write a RelationshipAttribute, which is owned by the Recipient, once **and** the **Sender proposes a value** which might make sense as a default.<br>Example: Company asks new customer to subscribe to the newsletter and proposes the subscription as default once. |
+| Type and context                                                                                        | Owner       | Possible? | Automation      | Remarks, reasons and examples                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Well observed, I thought about it for a long time. I want to keep the lines with "IdentityAttribute + Sender as owner" and "IdentityAttribute + third party as owner", although I said above already that the Recipient must be the `owner`, in order to specify the reason why the Sender or a third party cannot be the `owner`. It doesn't take up that much space either and thus provides an overview in the table that is as complete as possible and at the same time relatively simple. However, I left out the "RelationshipAttributes + third party" stuff to reduce complexity and because I already mentioned above in the text that this is not possible. 

I decided in the same way for the "Create Attributes for peer" scenario.

Reply:
I don't think removing the context is an option if it is wanted to ensure accuracy. I've thought about it again and think it's best to just do it the way you initially suggested and remove the "IdentityAttribute+ThirdPartyAsOwner"-row (for both mentioned guides). 👍